tests/md5/main.c

```diff
@@ -49,20 +49,20 @@ perftest(void)
 	for (i = 0; i < BLOCKCOUNT; i++)
 		MD5_Update(&ctx, buf, BLOCKLEN);
 	MD5_Final(hbuf, &ctx);
-	hexify(hbuf, hbuf_hex, 16);
```

Comment:
Did you want to keep the `hexify()` inside the timing loop?

Reply:
It's fast enough to be insignificant; but no, it doesn't belong inside the timing code.